docusaurus.config.js

```diff
@@ -80,6 +80,15 @@ const config = {
     ],
   ],
 
+  plugins: [
+    [
+      '@docusaurus/plugin-google-gtag',
+      {
+        trackingID: 'G-G856J3S8ZG',
+        anonymizeID: true,
```

Comment:
This should be anonymizeIP not anonymizeID

Reply:
Yeah Thanks for mentioning. I think it was changed due to intellisense. I will resolve this.